internal/config/cache.go

```diff
@@ -0,0 +1,130 @@
+package config
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	// deprecation messages
+	deprecatedMsgMemoryEnabled    = `'cache.memory.enabled' is deprecated and will be removed in a future version. Please use 'cache.backend' and 'cache.enabled' instead.`
+	deprecatedMsgMemoryExpiration = `'cache.memory.expiration' is deprecated and will be removed in a future version. Please use 'cache.ttl' instead.`
```

Comment:
what do you think about moving these constants to a `deprecated.go` file? Currently, these are the only deprecated config options, but there will likely be more in the future. would be nice to have them in the same place I think